app/assets/stylesheets/common/admin/staff_logs.scss

```diff
@@ -365,14 +365,32 @@ table.screened-ip-addresses {
   float: left;
 }
 
+.admin-watched-words {
+  .clear-all-row {
+    display: flex;
+    margin-top: 10px;
+
+    .btn.clear-all {
+      margin-left: auto;
```

Comment:
Could you use `justify-content: flex-end` to allow flexbox to align this item?

Reply:
Actually there are a few `floats` below - if we could remove them and use flex that would be ideal.